projects/addon-charts/components/line-days-chart/line-days-chart.component.ts

```diff
@@ -217,12 +230,14 @@ export class TuiLineDaysChart implements AfterViewInit {
                     ? month
                     : [
                           ...month,
-                          array[index + 1].find((day) => !Number.isNaN(day[1])) || DUMMY,
+                          array[index + 1]?.find((day) => !Number.isNaN(day[1])) || DUMMY,
                       ],
             );
     }
 
-    private getDay(index: number): TuiDay {
-        return this.value[index - this.value[0][0].day + 1][0];
+    private getDay(index: number): TuiDay | undefined {
+        const start = this.value[0]?.[0];
```

Comment:
Let's add `?.day || 0` here so the next line is simpler

Reply:
<img width="1121" alt="image" src="https://github.com/user-attachments/assets/9fa494ce-2c2c-4a6e-a94f-3d2360f278f0">